src/main/java/seedu/address/model/person/CustomDate.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Creates a CustomDate object based on DATE_FORMAT
+ */
+public class CustomDate {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Date should follow the following format DD/MM/YYYY (day-day/month-month/year-year-year-year)";
```

Comment:
I think the ones in bracket are not necessary as DD/MM/YYYY is a commonly used date format
```suggestion
            "Date should follow the following format DD/MM/YYYY.";
```